install_config/configuring_sdn.adoc

```diff
@@ -142,7 +143,39 @@ networkConfig:
 *redhat/openshift-ovs-multitenant* for the *ovs-multitenant* plug-in, or
 *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 <4> Service IP allocation for the cluster
-====
+
+Alternatively, you can create a pod network with multiple CIDR ranges by
+adding separate ranges into the `clusterNetworks` field with the range and the `hostSubnetLength`. 
+
+Multiple ranges can be used at once, and the range can be expanded. Pods can
+also be moved from one range to another. Pod allocations occur in the order
+listed, then when the range is full, move to the next on the list.
+
+.Configuring a pod network for multiple CIDRs
+[source,yaml]
+----
+networkConfig:
+  clusterNetworkCIDR: 10.128.0.0/14 <1>
+  clusterNetworks:
+  - cidr: 10.128.0.0/14 <2>
+    hostSubnetLength: 9 <3>
+  - cidr: 10.132.0.0/14
+    hostSubnetLength: 9
+  externalIPNetworkCIDRs: null
+  hostSubnetLength: 9
+  ingressIPNetworkCIDR: 172.29.0.0/16
+  networkPluginName: redhat/openshift-ovs-multitenant <4>
+  serviceNetworkCIDR: 172.30.0.0/16
+----
+<1> Cluster network for node IP allocation
+<2> Ensure the first `cidr` entry matches the `clusterNetworkCIDR` entry
```

Comment:
the hostSubneLength of the first entry will also have to match

Reply:
@JacobTanenbaum -- All the hostSubnetLengths need to match at the moment, right?